### PR TITLE
drainer: rtrim char type column in sql

### PR DESCRIPTION
**Pull request by cartersz**
### What problem does this PR solve?

Issue Number: ref #1164

### What is changed and how it works?
when table defined the char type column, rtrim function will be used in sql

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### Release note

- No release note

### CODE REVIEW

**Comment by cartersz** [comment]
@lichunzhu PTAL

File: pkg/loader/model.go [hunk 1]
```diff
@@ -270,14 +270,23 @@ func (dml *DML) buildOracleWhere(builder *strings.Builder, oracleHolderPos int)
 		if wargs[i] == nil || wargs[i] == "" {
 			builder.WriteString(escapeName(wnames[i]) + " IS NULL")
 		} else {
-			builder.WriteString(fmt.Sprintf("%s = :%d", escapeName(wnames[i]), pOracleHolderPos))
+			builder.WriteString(fmt.Sprintf("%s = :%d", dml.processOracleColumn(escapeName(wnames[i])), pOracleHolderPos))
```
Comment (lichunzhu) [inline pkg/loader/model.go:273]: I'm afraid `escapeName(wnames[i])` can't get correct `dataType` from `dml.info.dataTypeMap`

File: pkg/loader/model_test.go [hunk 2]
```diff
@@ -271,6 +271,59 @@ func (s *SQLSuite) TestUpdateMarkSQL(c *check.C) {
 	c.Assert(mock.ExpectationsWereMet(), check.IsNil)
 }
 
+func (s *SQLSuite) TestOracleUpdateSQLCharType(c *check.C) {
+	dml := DML{
+		Tp:       UpdateDMLType,
+		Database: "db",
+		Table:    "tbl",
+		Values: map[string]interface{}{
+			"ID":      123,
```
Comment (lichunzhu) [inline pkg/loader/model_test.go:280]: Add a column about escaped case?

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by gozssky — APPROVED** [review]

**Pull request merged**